### DISCUSSION TITLE
peter-bochs

### DISCUSSION THREAD

**Discussion by quantrpeter** [bochs-emu/Bochs#331] [General]
Hi All
    I have left bochs for 8 years and now coming back because I will use bochs to teach student. When I was developing peter-bochs I used pipeline to parse the bochs output, is there any upgrade these years? for example use xml/json as bochs output?
thanks
Peter

**Comment by stlintel** [comment]
Hi

What kind of output you mean ?
The screen output ? What is the point of converting it to XML ?
Instrumentation output ? You re feel free to create it in any format you want, even XML.

**Reply by quantrpeter** [reply]
peter-bochs is parsing the stdout from bochs.
